### ISSUE TITLE
Improved Spawn point display for offline mode

### ISSUE THREAD

**Issue by TheNewDarkOne** [RocketMap/RocketMap#1472]
When running server in offline mode (-os, ie no scanning). I have some suggestions to improve the map for using it as purely a spawn point map.
1. Ability to hide inactive spawn points (gray dots) while still showing the ones active soon (blue)
2. Ability to show times under spawn points for active ones before pokemon despawns.

the premise here is that you can use your already collected data to use your map as a navigation tool for spawn points even if you can't scan what's actually there. Especially when in between protocol versions/waiting on updates to pogoapi.

**Comment by matt-am** [comment]
This would be awesome. Using 4 really distinct colors would also help (I'm pretty color blind and right now the spawn point colors are just confusing)

**Comment by kbtbc** [comment]
These ideas would make great use of the map data while we don't have active scan options.   

It would be really great if we could also get some kind of Traveling Salesman pathing outputs to take us through best routes of spawnpoints.   Or more simple, allow time-shifting to view active spawns at a future time (ie. 5 or 15 or 45 minutes into the future) ... 

btw, I would also like to see time-stamps in the heat maps.  :-)

**Comment by mew905** [comment]
just delete the gray dot in static/icons. The map will 404 the dot in debug but the map itself will look the same. Only problem is it's still sending requests that do take time. Removing the code that grabs the grey dots makes *one* spawn point work, so I still havent figured out how to remove it completely

**Comment by sebastienvercammen** [comment]
Closing this issue because it's very old and no longer relevant to the current codebase. If the issue still persists for you today, comment on this issue to reopen it with new details (logs, screenshots).
